crates/rerun_c/src/rerun.h

```diff
@@ -198,6 +198,7 @@ enum {
     _RR_ERROR_CODE_CATEGORY_RECORDING_STREAM = 0x000000100,
     RR_ERROR_CODE_RECORDING_STREAM_CREATION_FAILURE,
     RR_ERROR_CODE_RECORDING_STREAM_SAVE_FAILURE,
+    RR_ERROR_CODE_RECORDING_STREAM_STDOUT_FAILURE,
```

Comment:
Also need to copy this into error.hpp at the right spot :(

Reply:
Huh, nice catch